webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubrics-manager.js

```diff
@@ -131,6 +140,133 @@ class SakaiRubricsManager extends RubricsElement {
       });
     });
   }
+
+  sortSiteRubrics(event) {
```

Comment:
sortSiteRubrics and sortSharedRubrics are very similar can common code be combined here?